packages/dx-react-grid/docs/reference/table-group-row.md

```diff
@@ -71,6 +73,24 @@ Field | Type | Description
 ------|------|------------
 row | [GroupRow](#grouprow) | The group row.
 
+### TableGroupRow.ContentProps
+
+Describes properties passed to a component that renders a group cell content.
+
+Field | Type | Description
+------|------|------------
+row | [GroupRow](#grouprow) | The group row.
+column | [Column](grid.md#column) | The column associated with the group.
+children? | ReactNode | A React node to be rendered within the cell's content.
+
+### TableGroupRow.IconProps
+
+Describes properties passed to a component that renders a group expand icon.
+
+Field | Type | Description
+------|------|------------
+expanded | boolean | Specifies whether the row is expanded.
```

Comment:
This is not required property

Reply:
Can't agree here: the only purpose of the Icon component is to display a state of this property

Reply:
In the `propTypes` this property is not required